paranoid_crypto/lib/randomness_tests/lattice_suite_test.py

```diff
@@ -47,9 +47,9 @@ def testFindBiasImpl(self, output_size: int):
     test_size = 100
     sample = []
     for _ in range(test_size):
-      gmpy.rand("init", output_size)
-      gmpy.rand("seed", int.from_bytes(os.urandom(16), "little"))
-      block = int(gmpy.rand("next", 1 << block_size))
+      # TODO: the default RNG has changed from LCG (that these tests assume) to MT, so tests will fail.
```

Comment:
Can we implement here the old GMPY LCG or use another one, so the tests pass? I'm not confortable with having the tests failing.

Reply:
> The main culprit I see with 1 and 2 is that GMP is GPL-licensed and therefore either of those options would be a no-go for Apache-licensed Paranoid. I am, of course, not a lawyer, but this seems to be a clear-cut case of licence incompatibility we probably don't want to go into.

Oh I wasn't aware of the license issues with 1 and 2. 

>  As far as I understand them, they should be working ok against anything of that sort, but I'm not sure and I'd appreciate you opinion based on better understanding of the method(s) used in the test set.

You are right, the tests should be able to identify similar LCGs and they usually have a common structure but with different constants and lengths. So in the end it doesn't need to be the same LCG used in gmpy. Maybe we can use or implement one with a similar structure and different constants.

Reply:
Hi Alex,

Any luck with giving option 3 a try? Thanks!